app/lib/methods/loadMissedMessages.ts

```diff
@@ -26,14 +28,14 @@ async function load({ rid: roomId, lastOpen }) {
 	return result;
 }
 
-export default function loadMissedMessages(args) {
+export default function loadMissedMessages(this: IRocketChat, args: { rid: string; lastOpen: string }): Promise<void> {
 	return new Promise(async (resolve, reject) => {
 		try {
 			const data = await load.call(this, { rid: args.rid, lastOpen: args.lastOpen });
-
 			if (data) {
-				const { updated, deleted } = data;
-				await updateMessages({ rid: args.rid, update: updated, remove: deleted });
+				const { updated, deleted }: { updated: ILastMessage[]; deleted: ILastMessage[] } = data;
+				// loaderItem is null only to surpass the obligatoriness of the item in the function, as soon as it is migrated it will not be necessary.
```

Comment:
WDYF about add an `TODO` with a `@ts-ignore`? 
I think this makes it easier not to forget to change it later